model/logs/mobile-crashes.yaml

```diff
@@ -0,0 +1,42 @@
+groups:
+  - id: mobile-crash-event
+    type: event
+    brief: >
+      This document defines attributes for crashes represented using Log Record Events.
+      `event.domain` is 'mobile' and `event.name` is 'crash
+    attributes:
+      - id: platform
+        requirement_level: "required"
+        brief: >
+            This attribute the platform type of the device on which the crash event occurred.
+        type:
+          allow_custom_values: false
+          members:
+            - id: android
+              value: 'android'
+              brief: 'Value if crash happened on android device'
+            - id: ios
+              value: 'ios'
+              brief: 'Value if crash happened on ios device'
+      - id: isAppInForeground
+        requirement_level: "required"
+        brief: >
+          This attribute indicates whether the app was in the foreground or not during the crash.
+        type:
+          allow_custom_values: false
+          members:
+            - id: true
+              value: 'true'
+              brief: 'Value if crash happened in foreground'
+            - id: false
+              value: 'false'
+              brief: 'Value if crash did not happen in foreground'
+      - id: data
+        requirement_level: "required"
+        type: attribute_group
+        brief: >
+          'event.data' in the case of mobile crashes will be an attribute that contains nested attributes.
```

Comment:
I think nested attributes haven't been approved into the OTel spec yet if I'm not mistaken, so if we wanted to use them we should probably add that dependency on this PR.

Another alternative (also not approved) would be supporting structured log bodies that would allow for nested properties. There's a prototype on it in the [Java repo](https://github.com/open-telemetry/opentelemetry-java/pull/5880) in case anyone's interested!

Reply:
I'm sure it's slightly contentious, but the _stable_ logs data model does [specify attributes](https://github.com/open-telemetry/opentelemetry-specification/blob/eda62e4baf906ac8225a9ee69fa97ff7f48dbccc/specification/logs/data-model.md#field-attributes) as `map<string,any>`. Contrast this with the [common attributes spec](https://github.com/open-telemetry/opentelemetry-specification/tree/eda62e4baf906ac8225a9ee69fa97ff7f48dbccc/specification/common#attribute) that constrains the values that attributes can have (primitives or homogeneous arrays only). 

The common attributes spec [discusses mapping arbitrary composite data](https://github.com/open-telemetry/opentelemetry-specification/blob/eda62e4baf906ac8225a9ee69fa97ff7f48dbccc/specification/common/attribute-type-mapping.md#composite-values) into attributes, so there's probably a case to be made.

Reply:
Why not making it explicit? I.e. reusing `exception.message`, `exception.stackstrace`, `thread.name` / `thread.id` attributes instead of encoding them into a `data` object?

Reply:
@tonzhan2  I suggest listing the contents of `event.data` as top level attributes to begin with, to get feedback. And on the side let's continue to ask how to represent nested attributes in semantic conventions.